2022/Day2/Day2.py

```diff
@@ -0,0 +1,44 @@
+file2 = open('input.txt', 'r')
+
+total_score = 0
+
+while True:
+    Line = file2.readline().strip()
+    if len(Line) == 0:
+        break
+    Line = Line.split(' ')
+
+    if Line[1] == 'X':
```

Comment:
As an exercise, I'd like you to try replacing this whole thing with a dictionary lookup. Something like
```
total_score += my_dict[something][something_else]
```
but you have to figure out what the two somethings are. That can be done in a follow-up PR, though.

Reply:
Will address this after this merge.